src/views/Resources.vue

```diff
@@ -2,15 +2,7 @@
   <main>
     <form class="search-form">
       <Input :value.sync="searchTerm" name="Search" placeholder="Search resources..." />
-      <Select :value.sync="selectedType" name="Resource Type">
-        <option value="">All Resource Types</option>
-        <option
-          :key="resourceType.id"
-          :value="resourceType.name"
-          v-for="resourceType in nonAbstractResourceTypes">
-          {{resourceType.name}}
-        </option>
-      </Select>
+      <Select :value.sync="selectedType" placeholder="All Resource Types" name="Resource Type" :options="nonAbstractResourceTypes" />
```

Comment:
This lines seems to be quite long :dromedary_camel:

Reply:
done